test/jdk/java/nio/file/WatchService/LotsOfEvents.java

```diff
@@ -106,6 +108,7 @@ static void drainAndCheckOverflowEvents(Path dir,
         boolean gotOverflow = false;
         while (key != null) {
             List<WatchEvent<?>> events = key.pollEvents();
+            System.out.println("Polling retrieved " + events.size() + " events");
```

Comment:
Minor nit, this should be "event(s)" to avoid "1 events".

Reply:
Done. Addressed in latest version of PR.